Makefile

```diff
@@ -0,0 +1,47 @@
+# Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+BINARY_NAME=nancy
+BUILD_VERSION_LOCATION=github.com/sonatype-nexus-community/nancy/buildversion
+GOLANGCI_LINT_DOCKER=golangci/golangci-lint:v1.23.1
+
+all: deps test lint build
+
+.PHONY: lint clean deps env-setup build test integration-test package
+
+lint:
+	docker run --rm -v $$(pwd):/app -v $$(pwd)/.cache:/root/.cache -w /app $(GOLANGCI_LINT_DOCKER) /bin/sh -c "golangci-lint cache status --color always && golangci-lint run --timeout 5m --color always -v"
+
+clean:
+	$(GOCLEAN)
+	rm -f $(BINARY_NAME)
+	rm -f $(BINARY_NAME)-*amd64*
+
+deps:
+	$(GOCMD) mod download
+	$(GOCMD) mod verify
+	$(GOCMD) mod tidy
+
+env-setup:
+	export GO111MODULE=on CGO_ENABLED=0
+
+build: env-setup
+	$(GOBUILD) -o $(BINARY_NAME) -v
+
+test:
+	$(GOTEST) -v ./...
```

Comment:
This may not really relate to this PR, but running tests with the race detector (i.e. `go test -race -v ./...`) is often a good idea, too, and is at least tangentially related to linting :)

Reply:
![race](https://user-images.githubusercontent.com/1099906/72849794-8812e000-3c75-11ea-9f32-a7db84be80bc.gif)

:) im up for `racing` ... added in commit c4fbd81